contact/style.scss

```
@@ -0,0 +1,348 @@
+body{
+  background-color: #FFF;
+  margin: 32px;
+}
+
+h1{
+  height: 2.2rem;
+  text-align: right;
+  height:54px;
+}
+
+.content-page-title{
+  color: #000000;
+  text-align: center;
+  hirgt:77px;
+  font-family: 'Hiragino Kaku Gothic ProN', 'ヒラギノ角ゴ ProN W3', sans-serif;
+}
+
+
+
+
+
+.contact-primary{
+  p{
+    color: #707070;
+    font-style:normal;
+    font-family: 'DNP 秀英明朝 Pr6', 'ヒラギノ角ゴ ProN W3', sans-serif;
+    }
+  &__mail{
+    position: relative;
+    width: 320px;
+    margin: 0 auto;
+  }
+  &__mail-icon-contents{
+    border-bottom: 1px solid #707070;
+    height: 51px;
+    width: 248px;
+    margin-top:52px;
+    margin-left: auto;
+    margin-right: auto;
+    clear: both;
+  }
+  &__mail-icon{
+    width: 40px;
+    margin: 8px;
+    float: left;
+    > img {
+      outline:0;
+      width: 37px;
+    }
+  }
+  &__mail-contents{
+    color: #707070;
+    margin: 8px;
+    float: left;
+    font-size: 2.2rem;
+    height: 40px;
+    width: 171px;
+    text-align: center;
+  }
+  &__mail-copy{
+    position: absolute;
+    width: 32px;
+    top: 20px;
+    right: 0;
+    background-color: transparent;
+    border: none;
+    > img {
+      width: 100%;
+    }
+  }
+
+  &__phone{
+    position: relative;
+    width: 320px;
+    margin: 0 auto;
+  }
+  &__phone-icon-contents{
+    border-bottom: 1px solid #707070;
+    height: 51px;
+    width: 248px;
+    margin-top:48px;
+    margin-left: auto;
+    margin-right: auto;
+    clear: both;
+  }
+  &__phone-icon{
+    width: 40px;
+    margin: 8px;
+    float: left;
+    > img {
+      outline:0;
+      width: 30px;
+    }
+  }
+  &__phone-contents{
+    color: #707070;
+    margin: 8px;
+    float: left;
+    font-size:2.2rem;
+    height: 40px;
+    width: 171px;
+    text-align: center;
+  }
+  &__phone-copy{
+    position: absolute;
+    width: 32px;
+    top: 20px;
+    right: 0;
+    background-color: transparent;
+    border: none;
+    > img {
+      width: 100%;
+    }
+
+}
+}
+
+.contact-tpl{
+  &__button-1st_wrapper{
+    height: 48px;
+    margin-top:24px;
+    margin-bottom:16px;
+    width: 100%;
+  }
+
+  &__description{
+    color: #707070;
+    float: left;
+    width: 100%;
+    margin-right: -96px;
+    padding-right: 96px;
+    box-sizing: border-box;
+    line-height:21px;
+  }
+  &__button-1st{ 
+    color: #707070;
+    border: solid 1px #707070;
+    background-color: #FFF;
+    border-radius: 98px;
+    margin-top:24px;
+    width: 96px;
+    height: 48px;
+    float: right;
+    margin-bottom:24px;
+  }
+  &__copy-text li{
+    color:#707070;
+    list-style: none;
+    line-height:25px;
+    clear: both;
+  }
+  &__button-2nd_wrapper{
+    clear: both;
+    margin-left:aut;
+    margin-right:aut;
```

```suggestion
    margin-left:auto;
    margin-right:auto;
```
aut -> auto に修正してください